.gitignore

```diff
@@ -3,13 +3,14 @@ data/*
 
 example/.ipynb_checkpoints/
 test/__pycache__/
+test/*.txt
```

Comment:
Nope, logs are exported to `data/outputs/data_pipeline.log` (which is already ignored since we ignore `outputs`). A logging test script that I wrote writes logs to `test` though.

Reply:
Ok got it - this is for tests